### PR TITLE
fix TokenCard component style

### LINKED ISSUE #27: The token list
image

-----------
Some hints for building components:
* Our Figma design is here: https://www.figma.com/file/XtReDuYYGDVQBv4JfzVG0g/Greenstand---Wallet?node-id=7%3A71
* A tutorial of how to use Cypress component tool to build components: https://github.com/Greenstand/treetracker-wallet-web#how-to-build-a-component
* We are using MaterialUI to build the app: https://material-ui.com/

### PR DESCRIPTION
**Pull request by YassinHussein**
Fixes #27 

![]Screenshot (278)

### CODE REVIEW

File: src/components/common/theme.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ export default createTheme({
     },
     secondary: {
       main: '#86C232',
+      lightGreen: '#f7fbf7',
```
Comment (dadiorchen) [inline src/components/common/theme.js:20]: Good!

**Comment by dadiorchen** [comment]
@YassinHussein this version is much better than before, please just resolve the conflict in the PR, then I will merge it.

It's about the conflict on package-lock.json. I guess you just need to replace your local one with the master version.

**Comment by YassinHussein** [comment]
ok, thank you

**Comment by dadiorchen** [comment]
@YassinHussein is there any problem with resolving the conflict? If true, I can help

**Comment by YassinHussein** [comment]
> @YassinHussein is there any problem with resolving the conflict? If true, I can help

thanks @dadiorchen. it's all good

**Comment by dadiorchen** [comment]
@YassinHussein thanks for you good job!

**Pull request merged**